projects/storefrontapp-e2e-cypress/cypress/sample-data/b2b-bulk-pricing.ts

```diff
@@ -1,7 +1,19 @@
 export const PRODUCT = '2221933';
 export const PRODUCT_NO_PRICING = '3881074';
 export const TEST_QUANTITY = '31';
+export const QUANTITY_FOR_25_DISCOUNT = '100';
```

Comment:
If any of the sample data or calculations change, the test will break. Therefore for assertions, we have been following this approach:
`cy.wait(`@${loginAlias}`).then((xhr) => {
      const data=xhr.response.body.xxx;
      //compare against the data instead of hardcoded values
});`

Reply:
for the quantity, it's the value user inputs. So I did not change that.
But i did change the totals, so no more hardcoded totals, i'm getting it from the response now.